packages/ragbits-core/tests/unit/utils/pyproject/testprojects/factory_project/pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+name = "factory_project"
+
+[tool.ragbits.core.default_llm_factories]
+"ragbits.core.llms.base.LLMType.TEXT" = "ragbits.core.llms.factory.simple_litellm_factory"
```

Comment:
Sorry that I haven't caught this in the previous round of reviews, but could the format be the same as in the ticket, i.e. by enum values:

```
[tool.ragbits.core.default_llm_factories]
text = "ragbits.core.llms.factory.simple_litellm_factory"
vision = "ragbits.core.llms.factory.simple_litellm_vision_factory"
structured_outputs = "ragbits.core.llms.factory.simple_litellm_vision_factory"
```

Having to specify full path for each key seems unnecessary and complicated for the users. We already know that each key is a value in `ragbits.core.llms.base.LLMType`.